packages/ui/src/modal/bid/BidModalRenderer.tsx

```diff
@@ -496,6 +531,7 @@ export const BidModalRenderer: FC<Props> = ({
     trait,
     quantity,
     feesBps,
```

Comment:
With the changes above we should be observing reservoirMarketplace and not just the orderKind